app/assets/stylesheets/articles.scss

```diff
@@ -110,7 +110,7 @@
 
 @keyframes fade-in {
   0% {
-    opacity: 0;
+    opacity: 0.00001;
```

Comment:
small change (blocking) - as we'll be trying to make sure we always use this value instead of `opacity: 0`, let's add a variable for it in `_variables.scss`, e.g. `$opacity-0: 0.0001`. Then we can use it in all the places implemented here

Reply:
fixed